build.gradle

```diff
@@ -603,8 +684,8 @@ if (jdkVersion == '8') {
       compile project(':iceberg-orc')
       compile project(':iceberg-parquet')
       compile project(':iceberg-arrow')
-      compile project(':iceberg-hive-metastore')
-      compile project(':iceberg-spark')
+      compile project(':iceberg-hive2-metastore')
+      compile project(":iceberg-spark")
```

Comment:
Small nit: why the change from single quotes?

Reply:
Yes, it was unintentional. Thanks for the catch